es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="my-5 container">
```

Comment:
For each of the spacing classes (things like `my`, `mb`, `pb`), let's also change these to updated design system classes - should be a direct mapping from the "Deprecated Spacing" section [here](https://design.energysage.dev/atoms/spacing)